db/migrate/20170528102055_add_limits_resources_to_containers.rb

```diff
@@ -0,0 +1,8 @@
+class AddLimitsResourcesToContainers < ActiveRecord::Migration[5.0]
+  def change
+    add_column :containers, :request_cpu,    :float  # cores
+    add_column :containers, :request_memory, :bigint # bytes
```

Comment:
Should the column_names have a `_cores` or `_bytes` suffix to eliminate future confusion?